### PR TITLE
+htp akka http Throughput benchmark, for jenkins

### PR DESCRIPTION
**Pull request by ktoso**
For graphs on jenkins

### CODE REVIEW

**Comment by ktoso** [comment]
Installing `ab` on workers is in progress (spawned chef and it's going though all servers now).
Otherwise `wrk` is already there and that has the more interesting metrics for us

**Comment by ktoso** [comment]
This is step 1, step 2 will be moving this such that load generator runs on different node than the server. I'll have that done today as well I hope

**Comment by ktoso** [comment]
Please review @johanandren

File: akka-http-tests/src/test/scala/akka/http/scaladsl/server/AkkaHttpServerThroughputSpec.scala [hunk 1]
```diff
@@ -0,0 +1,225 @@
...
+      case Some(false) ⇒ throw new TestPendingException()
+      case Some(true)  ⇒ test
+      case None ⇒
+        import scala.sys.process._
+
+        val wrk = """ab -h""".!
+        _ifAbAvailable = Some(wrk == 22) // app found, help displayed
+        ifAbAvailable(test)
+    }
+  }
```
Comment (johanandren) [inline akka-http-tests/src/test/scala/akka/http/scaladsl/server/AkkaHttpServerThroughputSpec.scala]: This whole block, uhm wat? 
`wrk` -> `ab`
` _ifAbAvailable` -> `abAvailable`
Return code 22??

**Comment by johanandren** [comment]
Forking out to the shell, what's not to like. LGTM with some updated names in the forking methods.

File: akka-http-tests/src/test/scala/akka/http/scaladsl/server/AkkaHttpServerThroughputSpec.scala (hunk 1 above)
Comment (johanandren) [inline akka-http-tests/src/test/scala/akka/http/scaladsl/server/AkkaHttpServerThroughputSpec.scala]: Also it took a while to grokk why it's tailrec.

Comment (ktoso) [inline akka-http-tests/src/test/scala/akka/http/scaladsl/server/AkkaHttpServerThroughputSpec.scala]: yeah 22 is a weird return code `ab` gives when it prints help... I'll comment on it

**Pull request merged**